src/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-standard-components",
-  "version": "13.1.2",
+  "version": "13.1.3",
```

```suggestion
  "version": "13.2.0",
```
New feature, rather than bugfix, no?

---

doh. yes. ok.